src/routes/tools/annual-averages/ExploreData.svelte

```diff
@@ -54,14 +55,38 @@
   let showShare = false;
   let showChangeLocation = false;
 
-  $: metadata = [
-    ["boundary", $boundary.id],
-    ["feature", $location.title],
-    ["center", `${$location.center[0]}, ${$location.center[1]}`],
-    ["scenario", $scenario.label],
-    ["climate indicator", $climvar.label],
-    ["units", $climvar.units.imperial],
-  ];
+  let ChangeLocation;
+  let DownloadChart;
+
+  let metadata;
+  let csvData;
+  let printContainer;
+  let printSkipElements;
+
+  function loadLocation() {
+    showChangeLocation = true;
+    import("../../../components/tools/Partials/ChangeLocation.svelte").then(
```

Comment:
I think you can also use `async/await` here:

```js
  async function loadLocation() {
    showChangeLocation = true;
    ChangeLocation = (
      await import("../../../components/tools/Partials/ChangeLocation.svelte")
    ).default;
  }
```

Reply:
Changed the units to a hanging punctuation. This format did not work for earlier iterations because the box had > 2 values, and longer units like `inches` & values with more significant digits. Will need to keep an eye on it for future tools.

Reply:
That makes sense, thanks for pointing that out to me.